bin/garnett2pos

```diff
@@ -1,233 +0,0 @@
-#!/usr/bin/env python
```

Comment:
I'm not clear on how the removal of this file is related dropping Py27 support and what the motivation in the first place is.

Reply:
I'm also confused about why this is being removed. I find this script quite helpful (which is why I wrote/included it). I think it would be helpful to refactor this file into a `__main__.py` that can convert files from the command line, with a command line entry point like `signac` has, but that's a separate issue.

Reply:
I can't recall why I deleted this, so I will revert it

Reply:
> that can convert files from the command line, with a command line entry point like `signac` has, but that's a sep

I support this idea